### ISSUE TITLE
ENA-229: Output connection location can be confusing when dragging long reporters

### ISSUE THREAD

**Issue by msucharski** [LLK/scratch-blocks#1156]
### Expected Behavior

When a user drags an input block, they should be able to drop the block into the insertion point when they release their mouse over the insertion point. 

### Actual Behavior

The left coordinate of the input block is used to determine what insertion marker/input "hole" to place the dragged block. This is confusing, especially on longer input blocks where the center or grabbed part of the block is far from the left edge of the block. 

### Steps to Reproduce

Drag any longish input and try to place it into another block. Notice the selection doesnt use the center of the block or the mouse position, but instead uses the left edge. 

![image](https://user-images.githubusercontent.com/32305014/31784569-32127f6e-b4d0-11e7-8061-9cbcd053678c.png)

![image](https://user-images.githubusercontent.com/32305014/31784578-39d7a198-b4d0-11e7-896a-eda0fdd53dee.png)

### Operating System and Browser

Any OS, any browser

**Comment by mrjacobbloom** [comment]
related to #456

**Comment by Wowfunhappy** [comment]
Hi! Is there a reason the "help wanted" label was removed? My students find this quite confusing!

**Comment by cwillisf** [comment]
Hello! I'm sorry your students find this confusing. We removed the "help wanted" label because this design decision can have many complex and subtle implications. We'd like to address the issue, but the exact solution we go with may or may not be the one suggested here.

With any change to interaction, we'll need to consider how this change might affect touchscreen users, touchpad users, and mouse users, as well as both left-to-right and right-to-left languages. For this issue in particular, if the destination block has multiple inputs, the pointer or the center of the dragged block might be over one input, while either edge might be over a different input -- maybe even both. In this case, any input(s) other than one(s) at the edge(s) could be obscured from view, so choosing one of them could lead to a different sort of confusion.

Because we haven't yet committed to one specific solution, we're not ready to mark this "help wanted" yet.
